contrib/drivers/filesharedrivers/nfsnative/nfsnative.go

```diff
@@ -0,0 +1,87 @@
+package nfsnative
+
+import (
+	//"github.com/ceph/go-ceph/rados"
+	//"github.com/ceph/go-ceph/rbd"
+	log "github.com/golang/glog"
+	. "github.com/opensds/opensds/contrib/drivers/utils/config"
+	"github.com/opensds/opensds/pkg/model"
+	pb "github.com/opensds/opensds/pkg/model/fileshareproto"
+	"github.com/opensds/opensds/pkg/utils/config"
+	. "github.com/opensds/opensds/testutils/collection"
+)
+
+const (
+	opensdsPrefix   = "opensds-"
+	sizeShiftBit    = 30
+	defaultConfPath = "/etc/opensds/driver/nfsnative.yaml"
+	defaultAZ       = "default"
+)
+
+const (
+	KPoolName  = "NFSNativePoolName"
+	KImageName = "NFSNativeImageName"
+)
+
+type NFSNativeConfig struct {
+	ConfigFile string                    `yaml:"configFile,omitempty"`
+	//Pool       map[string]PoolProperties `yaml:"pool,flow"`
+}
+
+func EncodeName(id string) string {
+	return opensdsPrefix + id
+}
+
+/*func NewSrcMgr(conf *NFSNativeConfig) *SrcMgr {
+	return &SrcMgr{conf: conf}
+}*/
+
+/*
+type SrcMgr struct {
+	conn  *rados.Conn
+	ioctx *rados.IOContext
+	img   *rbd.Image
+	conf  *CephConfig
+}*/
```

Comment:
Please remove all code that is not needed and commented out.

Reply:
Ok sure i am closing this PR as it rebase issue, i am raising new PR for controller and dock implementation